src/commands/build.ts

```diff
@@ -315,7 +315,8 @@ export async function command(commandOptions: CommandOptions) {
               return spec;
             }
             if (dependencyImportMap.imports[spec]) {
-              let resolvedImport = path.posix.resolve(
+              let resolvedImport = path.posix.join(
```

Comment:
This seems fine to me. Not sure why we were using `resolve()` before, since we seem to just be concatening, not resolving its on-disk location.